### ISSUE TITLE
CI dependencies are broken

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#38067]
See test PR #38065 
https://drone.owncloud.com/owncloud/core/27424/1/4
```
behat composer.lock is not up to date.
composer bin behat install --no-progress
No lock file found. Updating dependencies instead of installing from lock file. Use composer update over composer install if you do not have a lock file.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - sensiolabs/behat-page-object-extension[v2.3.0, ..., v2.3.2] require ocramius/proxy-manager ^2.1.1 -> satisfiable by ocramius/proxy-manager[2.1.1, ..., 2.9.1].
    - ocramius/proxy-manager[2.2.0, ..., 2.2.3] require ocramius/package-versions ^1.1.3 -> satisfiable by ocramius/package-versions[1.1.3, ..., 1.11.0].
    - ocramius/proxy-manager 2.1.1 requires ocramius/package-versions ^1.1.1 -> satisfiable by ocramius/package-versions[1.1.1, ..., 1.11.0].
    - ocramius/package-versions[1.1.1, ..., 1.2.0] require composer-plugin-api ^1.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - ocramius/package-versions[1.3.0, ..., 1.4.2] require composer-plugin-api ^1.0.0 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - ocramius/package-versions[1.5.0, ..., 1.5.1] require php ^7.3.0 -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/package-versions[1.6.0, ..., 1.9.0] require php ^7.4.0 -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/proxy-manager[2.3.0, ..., 2.5.2] require php ^7.4.0 -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/package-versions[1.10.0, ..., 1.11.0] require php ^7.4.7 -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/proxy-manager[2.6.0, ..., 2.7.0] require php 7.4.* -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/proxy-manager 2.7.1 requires php 7.4.0 -> your php version (7.2.34) does not satisfy that requirement.
    - ocramius/proxy-manager[2.8.0, ..., 2.9.1] require php ~7.4.1 -> your php version (7.2.34) does not satisfy that requirement.
    - Root composer.json requires sensiolabs/behat-page-object-extension ^2.3 -> satisfiable by sensiolabs/behat-page-object-extension[v2.3.0, v2.3.1, v2.3.2].

ocramius/package-versions only provides support for Composer 2 in 1.8+, which requires PHP 7.4.
If you can not upgrade PHP you can require composer/package-versions-deprecated to resolve this with PHP 7.0+.

You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
make: *** [vendor-bin/behat/vendor] Error 2
```

**Comment by phil-davis** [comment]
This seems to be related to composer 2.0, which was released around 2020-10-24. I ran the owncloud-ci/php drone today, to pickup the latest patch releases of PHP 7.4 and 7.3 that came out a few days ago. I guess that has also picked up composer 2.0.

Most things seem OK, but behat is not working with PHP 7.2 or 7.3. I installed composer 2.0 locally on Ubuntu 18.04 with:
```
sudo composer self-update --2
```

And then I get the same dependency problems as are seen in CI.
